### ISSUE TITLE
Playground: Support multi-line expressions

### ISSUE THREAD

**Issue by aiksiongkoh** [Small-JS/SmallJS#19]
I went to https://small-js.org/Playground/Playground.html
Entered:
| a |
a := 10 factorial.

Clicked Button >Evaluate
Got message:
Error: Error: Compile error in file: , line: 4, column: 9: Illegal literal: |

I want to evaluate multi-line expression.

**Comment by FunctionPoint** [comment]
Hi there, thanks for the feedback.

The Playground is currently designed to compile single ST expressions (object message sends).
So the error message is intended behavior.

But it is certainly possible to change it to compile the input to method body,
This would require a compiler change that returns the last expression as the result.
(Normally it would return `self`, being an instance of `Evaluator`.)

I'll mark this as a feature request and change the title a bit to reflect that.

**Comment by FunctionPoint** [comment]
Turned out to be a minor change, thanks @aiksiongkoh  .

**Comment by FunctionPoint** [comment]
PS
The new Playground is now uploaded to the website https://small-js.org .

When testing in Chome, it still gave the wrong result due do cached compiled JS.
Pressing [Ctrl]-[F5] or [Shift]-[F5] was not enough to clear the cache.
I had to go to: `Settings` | `Delete Browsing Data`  and select `Cached images and files'.

In Edge, I had to do the same.

Firefox was OK after just pressing [Ctrl]-[F5].
